docs/project/commit_access.md

```diff
@@ -0,0 +1,64 @@
+# Commit access
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Getting access](#getting-access)
+-   [Removing access](#removing-access)
+
+<!-- tocstop -->
+
+## Overview
+
+Commit access is what allows developers to merge PRs into the main
+[carbon-lang repository](https://github.com/carbon-language/carbon-lang/). It
+also gives the ability to [review, approve, and merge changes](code_review.md).
+
+Developers with commit access are not required to do reviews. It is granted as a
+convenience in order to allow fixing PRs after approval, for example because the
+review noted a minor typo, or because there is a conflict that the PR author
+needs to resolve before merging.
+
```

Comment:
What do you think about skipping this paragraph? Not sure... But maybe the next paragraph (maybe with my suggestion) can suffice?

Reply:
In the past, everyone discussion I've had about commit access involved people mentioning they're not fully comfortable doing reviews. That's why it's the one thing I'd been focused on. Do you still want it removed?

Reply:
Interesting! No, I think that's a good reason to keep it. Maybe some slight word smithing?

```suggestion
Note that developers with commit access are not required to do reviews or help with approving and merging PRs. It can still be useful to allow authors to fix trivial PR comments after approval and then merge their own PR, for example because the
review noted a minor typo, or to resolve merge conflicts.

```